policyengine_us/variables/gov/states/id/tax/income/deductions/id_retirement_benefits.py

```diff
@@ -0,0 +1,38 @@
+from policyengine_us.model_api import *
+
+
+class id_retirement_benefits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho retirement benefits"
+    unit = USD
+    documentation = "https://tax.idaho.gov/wp-content/uploads/forms/EFO00088/EFO00088_12-30-2022.pdf#page=1"
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        head = person("is_tax_unit_head", period)
+        p = parameters(period).gov.states.id.tax.income.deductions.retirement_benefits
+        filing_status = tax_unit("filing_status", period)
+        
+        # check if filer is eligible for the deduction
+        age_threshold = p.age_eligibility
+        age_threshold_disabled = p.age_eligibility_disabled
+        disabled_head = tax_unit("disabled_head", period)
+        eligible = where(disabled_head == False, 
```

Comment:
need to create a id_retirement_benefits_eligible.py (bool) where we check for the eligibility

Reply:
need a unit test for the new file